bin/lib/c.js

```diff
@@ -64,9 +65,48 @@ const throwErr = (err) => {
 
 };
 
+/**
+ * Given an error, report it.
+ * @param  {Error}  err          The Error object.
+ * @param  {Object} program      The commander.js program.
+ * @return {void}
+ */
+const reportError = (err, program) => {
+
+    // eslint-disable-next-line no-console
+    console.error(`\n${red('Error:')} ${err.message}\n`);
+
+    /* eslint-disable padded-blocks */
+    if (program) {
+        program.help();
+    }
+    /* eslint-enable padded-blocks */
+
+};
+
+const composeUp = (service) => {
+
+    // We're bringing up just one container.
+    let upCommand = 'docker-compose up -d';
+
+    // We need to scale some containers.
+    /* eslint-disable padded-blocks */
+    if (['app', 'consul'].includes(service)) {
+        upCommand += ` --scale ${service}=2`;
```

Comment:
Should the number of services be configurable?

Reply:
@allanchau, yeah, I intend support this through some `json` in the `devops` folder. I'll implement this in another PR.